Makefile

```diff
@@ -129,11 +137,18 @@ uninstall-identity-scripts:
 	done
 	-rmdir -p $(prefix)$(datadir)/mender/identity
 
-uninstall-inventory-scripts:
+uninstall-inventory-scripts: uninstall-inventory-local-scripts uninstall-inventory-network-scripts
+	-rmdir -p $(prefix)$(datadir)/mender/inventory
+
+uninstall-inventory-local-scripts:
 	for script in $(INVENTORY_SCRIPTS); do \
 		rm -f $(prefix)$(datadir)/mender/inventory/$$(basename $$script); \
 	done
-	-rmdir -p $(prefix)$(datadir)/mender/inventory
```

Comment:
This should stay, no? And it should be added below too.

Reply:
The thinking went like:
```
uninstall-inventory-scripts: uninstall-inventory-local-scripts uninstall-inventory-network-scripts
	-rmdir -p $(prefix)$(datadir)/mender/inventory
```
This uninstalls all the inventory scripts, like it says.

The `uninstall-inventory-local-scripts` and `uninstall-inventory-network-scripts` are a subset of the inventory scripts installed. And hence, uninstalling only one of them, may actually leave scripts behind. Thus, this should not remove the directory, as then it would remove more scripts, than what the target actually installs, and would hence not be symmetric.

Reply:
Yes, but `rmdir` is "soft", it will not remove the directory if it's not empty.

Reply:
Ahh, I see. Kk. Adding.